### ISSUE TITLE
Vega chart does not render when re-run

### ISSUE THREAD

**Issue by rathboma** [polynote/polynote#916]
**Description**
After creating a (good looking) chart with the chart-builder I refresh the vega cell and the chart is blank.

**Reproduction**
1. write a sql cell
2. Click 'chart' button, build a simple chart
3. Run the vega cell again

**Environment**
Polynote version 0.3.8
Running on CentOS 7
Local Spark
HDFS cluster
JVM Version 1.8
Python: 3.6.2

How I installed: Downloaded, pip install, configured spark, ran using `python polynote.py`

**Configuration**
Only changes are setting spark to local master

**Notebook**
If applicable, please attach or paste the notebook file (`.ipynb`) you were trying to run when you see this bug. 

**Logs**
I don't see any log output related to this.

**Screenshots**
![Kapture 2020-06-03 at 16 25 45](https://user-images.githubusercontent.com/279769/83690749-f5853480-a5b6-11ea-8de7-ddbe9cbe3cb9.gif)

**Comment by rathboma** [comment]
I just upgraded to 0.3.10 and have the same issue

**Comment by jeremyrsmith** [comment]
@rathboma Yep, we've seen this issue too. The underlying problem is that `bigint` (e.g. `Long`) data types have a workaround so that they work in the plot editor, but that workaround isn't applied when they're run with the run button. We'll try to have it fixed in the next release, but for now you can workaround by casting to 32-bit int (if possible) or 64-bit float.

**Comment by jeremyrsmith** [comment]
(the underlying problem being that 64-bit integers turn into `BigInt` on the browser-side, but Vega doesn't support those.)

**Comment by rathboma** [comment]
so the weird thing is that it works if I change to using `mean` rather than sum. My values are `int8`, which I'm assuming is not a 64bit integer?

**Comment by jeremyrsmith** [comment]
`int8` is in fact 8-byte integer, which is a 64-bit integer. `mean` casts it to `float8`, because `mean` requires division.
